Tasks/DeployVisualStudioTestAgent/make.json

```diff
@@ -8,7 +8,7 @@
         ],
         "files": [
             {
-                "url": "https://testexecution.blob.core.windows.net/testexecution/3840309/TestExecution.zip",
```

Comment:
Are we planning to update the additional dependencies for 125 today? This might get picked up for today's hotfix to Ring 0.